### DISCUSSION TITLE
21-06-14 일일회고

### DISCUSSION THREAD

**Discussion by GleamingStar** [d-h-k/issue-tracker#58] [회고]
또 늦었습니다 🤦‍♂️

**Comment by GleamingStar** [comment]
오늘은 DetailIssue 컴포넌트를 구현했습니다!
![image](https://user-images.githubusercontent.com/70461368/121869272-d6c68b80-cd3c-11eb-9015-eab50d634a18.png)
내일부터는 모양만 있고 기능이 없는 친구들을 API와 결합해 기능을 추가할 생각입니다!
마침 오프라인 가는 날이라 스타브도 오신다면 실시간 피드백 👍

**Comment by eeeesong** [comment]
오늘은 지난주 리팩토링한 부분을 잭슨과 함께 한번 더 리뷰하고 레이블 목록 부분 UI를 만들었습니다~
감기기운이 좀 있어서 아침 스크럼 참여도 못하고 컨디션이 좋지 않았는데, 이제 좀 회복이 된 것 같습니다 :) 
내일은 레이블 부분 API 연동을 해보고, hex를 컬러로 변환하는 모델을 작성할 것 같습니다~
